src/Components/Facility/LocationManagement.tsx

```diff
@@ -289,7 +289,7 @@ const Location = ({
         border
         className={cn(
           "mt-3 flex w-full items-center justify-between",
-          totalBeds != null && "opacity-50",
+          totalBeds != null && "opacity-100",
```

Comment:
```suggestion
          totalBeds == null && "opacity-50",
```

Reply:
if opacity-50 manage bed button will greyed out

Reply:
I think we should disable the button, why hack the class? Set disabled: true?